superset-frontend/src/cccs-viz/plugins/hooks/useEmitGlobalFilter.tsx

```diff
@@ -86,6 +112,14 @@ const useEmitGlobalFilter = () => {
             .filter(f => {
               const adhoc_filters = (filter.extraFormData?.adhoc_filters ||
                 []) as any[];
+              if (colData?.[f.subject]?.isDateColumn) {
+                dispatch(
+                  addWarningToast(
+                    'Date/Time columns cannot be filtered on selection. Use the filter bar to select a date range',
```

Comment:
I think we could support that, it seems to work ok at the moment, it's using the IN operator, the query is fine, the only thing is the values shown in the filter section, they are numeric.

Reply:
filter bar is just the native filters side bar (don't have a better name for it 😆 ). we could also remove this warning and limitation because like @cccs-joel said the query works fine and it works the same as the emit filters feature.